resources/lang/vi/authorization.php

```diff
@@ -21,134 +21,134 @@
 return [
     'beatmap_discussion' => [
         'destroy' => [
-            'is_hype' => 'Can not undo hyping.',
-            'has_reply' => 'Can not delete discussion with replies',
+            'is_hype' => 'Không thể hoản lại hype.',
```

Comment:
`hoãn` chứ ko phải là `hoản`
Nên dịch là `không thể tắt hype` , nếu có từ nào tốt hơn tắt thì dùng cũng đc

Not-so-good-rational explaination:
undo => irreversible => không đảo ngược đc (hype) =>  (chỉ có tắt/mở hype) => tắt

Reply:
À typo chữ `hoàn` ấy mà :3